### PR TITLE
Sortirovka

### PR DESCRIPTION
**Pull request by MateAcademy**

### CODE REVIEW

File: src/main/java/mate/academy/hw3/Sortirivka.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+        int[] result = mergeArrays(a1,a2);
+
+        for (int a : result) {
+            System.out.print(a + " ");
+        }
+    }
+
+    public static int[] mergeArrays(int[] a1, int[] a2) {
+        int[] a3 = new int[a1.length + a2.length];
```
Comment (Siril-O) [inline src/main/java/mate/academy/hw3/Sortirivka.java:17]: rename a3

File: src/main/java/mate/academy/hw3/Sortirivka.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        int i=0, j=0;
```
Comment (Siril-O) [inline src/main/java/mate/academy/hw3/Sortirivka.java:19]: rename i j

File: src/main/java/mate/academy/hw3/Sortirivka.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package mate.academy.hw3;
+
+public class Sortirivka {
```
Comment (Siril-O) [inline src/main/java/mate/academy/hw3/Sortirivka.java:3]: Sortirivka  is not valid name

**Review by Siril-O — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/hw3/Sortirivka.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                a3[k] = a;
+                i++;
+            }
+            else if (a1[i] < a2[j]) {
+                int a = a1[i];
+                a3[k] = a;
+                i++;
+            }
+            else {
+                int b = a2[j];
```
Comment (Siril-O) [inline src/main/java/mate/academy/hw3/Sortirivka.java:38]: why do you need b here

File: src/main/java/mate/academy/hw3/Sortirivka.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                a3[k] = a;
+                j++;
+            }
+            else if (j > a2.length-1) {
+                int a = a1[i];
+                a3[k] = a;
+                i++;
+            }
+            else if (a1[i] < a2[j]) {
+                int a = a1[i];
```
Comment (Siril-O) [inline src/main/java/mate/academy/hw3/Sortirivka.java:33]: why do you need a here

File: src/main/java/mate/academy/hw3/Sortirivka.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    public static int[] mergeArrays(int[] a1, int[] a2) {
+        int[] a3 = new int[a1.length + a2.length];
+
+        int i=0, j=0;
+        for (int k=0; k<a3.length; k++) {
+
+            if (i > a1.length-1) {
+                int a = a2[j];
```
Comment (Siril-O) [inline src/main/java/mate/academy/hw3/Sortirivka.java:23]: why do you need a here

**Review by Siril-O — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/hw3/Sortirivka.java (hunk 1 above)
Comment (MateAcademy) [reply src/main/java/mate/academy/hw3/Sortirivka.java:17]: https://github.com/MateAcademy/lessons/pull/16/commits/c3fde5ed713554b78d0b4d63bb86915deef9dfd4

File: src/main/java/mate/academy/hw3/Sortirivka.java (hunk 2 above)
Comment (MateAcademy) [reply src/main/java/mate/academy/hw3/Sortirivka.java:19]: https://github.com/MateAcademy/lessons/pull/16/commits/abb13fd80c57622a18c379c1558c5e8df71fd998

File: src/main/java/mate/academy/hw3/Sortirivka.java (hunk 3 above)
Comment (MateAcademy) [reply src/main/java/mate/academy/hw3/Sortirivka.java:3]: https://github.com/MateAcademy/lessons/pull/16/commits/abb13fd80c57622a18c379c1558c5e8df71fd998

File: src/main/java/mate/academy/hw3/Sortirivka.java (hunk 4 above)
Comment (MateAcademy) [reply src/main/java/mate/academy/hw3/Sortirivka.java:38]: https://github.com/MateAcademy/lessons/pull/16/commits/abb13fd80c57622a18c379c1558c5e8df71fd998

File: src/main/java/mate/academy/hw3/Sortirivka.java (hunk 5 above)
Comment (MateAcademy) [reply src/main/java/mate/academy/hw3/Sortirivka.java:33]: https://github.com/MateAcademy/lessons/pull/16/commits/abb13fd80c57622a18c379c1558c5e8df71fd998

File: src/main/java/mate/academy/hw3/Sortirivka.java (hunk 6 above)
Comment (MateAcademy) [reply src/main/java/mate/academy/hw3/Sortirivka.java:23]: https://github.com/MateAcademy/lessons/pull/16/commits/abb13fd80c57622a18c379c1558c5e8df71fd998

**Pull request merged**